tests/benchmark/compute/instruction/test_block_context.py

```diff
@@ -36,14 +36,29 @@ def test_block_context_ops(
     )
 
 
+@pytest.mark.parametrize(
+    "index",
+    [
+        0,
+        1,
+        256,
+        257,
+        None,
```

Comment:
```suggestion
        pytest.param(None, id="random"),
```
Or similar to show a better test id.